app/clients/ethereum/client_test.go

```diff
@@ -0,0 +1,198 @@
+/*
+* Copyright 2021 LimeChain Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package ethereum
+
+import (
+	"fmt"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	tc "github.com/limechain/hedera-eth-bridge-validator/test/test-config"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"math/big"
+	"testing"
+)
+
+func TestNewClient(t *testing.T) {
+	nodeClient := NewClient(tc.TestConfig.Validator.Clients.Ethereum)
+	assert.NotNil(t, nodeClient)
+}
+
+func TestGetClient(t *testing.T) {
+	nodeClient := NewClient(tc.TestConfig.Validator.Clients.Ethereum)
+	assert.NotNil(t, nodeClient)
+
+	client := nodeClient.GetClient()
+	assert.NotNil(t, client)
+}
```

Comment:
You can initialize a "mocked" client as a global variable and use it as expectation when asserting the client returned from `NewClient` and `GetClient`